.github/workflows/cli.yml

```diff
@@ -149,7 +149,8 @@ jobs:
         with:
           lint: false
 
-      - name: Build release
+      - if: always()
```

Comment:
What's with the `always()`s?

Reply:
So that we get a binary uploaded even if the sanitisation fails. But I only needed it temporarily while the tests were failing, so I'll remove it now